fitbenchmarking/fitbenchmarking
# Specify minimizers in regression tests

#### Description of Work
This PR ensures that we choose specific minimizers to test against for each software (i.e. the most-used minimizers rather than just the first alphabetical minimizer).

Fixes #946

#### Testing Instructions

1. Make sure the regression tests pass, and that the new results look sensible.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
